schema.development.sql

```diff
@@ -1036,14 +1047,30 @@ CREATE OR REPLACE ACTION get_access_grants_granted ($page INT, $size INT) PUBLIC
 
     $offset int := $index * $limit;
 
-    return SELECT id, ag_owner_user_id, ag_grantee_wallet_identifier, data_id, locked_until, content_hash, inserter_type, inserter_id FROM access_grants
-        WHERE ag_grantee_wallet_identifier =  @caller COLLATE NOCASE ORDER BY height ASC, id ASC LIMIT $limit OFFSET $offset;
+    if $user_id is null {
+      return SELECT id, ag_owner_user_id, ag_grantee_wallet_identifier, data_id, locked_until, content_hash, inserter_type, inserter_id
+        FROM access_grants
+        WHERE ag_grantee_wallet_identifier = @caller COLLATE NOCASE
+        ORDER BY height ASC, id ASC LIMIT $limit OFFSET $offset;
+    } else {
+      return SELECT id, ag_owner_user_id, ag_grantee_wallet_identifier, data_id, locked_until, content_hash, inserter_type, inserter_id
+        FROM access_grants
+        WHERE ag_owner_user_id = $user_id AND ag_grantee_wallet_identifier = @caller COLLATE NOCASE
```

Comment:
Just to make the difference obvious.
```suggestion
        WHERE ag_owner_user_id = $user_id
          AND ag_grantee_wallet_identifier = @caller COLLATE NOCASE
```